docs/src/guide/installation.md

```diff
@@ -32,145 +32,269 @@ Please help us [keep these instructions up-to-date][contribute]!
-	pip3 install modm gdbgui
+```sh
+sudo apt install python3 python3-pip scons git
+pip3 install modm
+```
+
+We recommend using a graphical frontend for GDB called [gdbgui][]:
+
+```sh
+pip3 install gdbgui
+```
 
 !!! info "Python Packages in PATH"
 	The pip3 command installs executables into `~/.local/bin`, which
 	must be added to PATH if it is not already the case.
 	Add the following line to the end of your `~/.bashrc` file:
 
-		export PATH="~/.local/bin:$PATH"
+	```sh
+	export PATH="~/.local/bin:$PATH"
+	```
 
-For host system (x86_64/arm64) targets:
+!!! warning "Ubuntu ≤ 19.10"
+	Ubuntu ≤ 19.10 unfortunately defaults to Python 2, however, our SCons tools
+	require Python 3. For a less intrusive way to run all scons scripts with
+	Python 3 add this to your `.bashrc` or `.zshrc`:
 
-	sudo apt install gcc build-essential libboost-all-dev
+	```sh
+	alias scons="/usr/bin/env python3 $(which scons)"
+	```
 
-For ARM Cortex-M targets:
+	If you get errors about missing `pyelftools`, check that you're really
+	running SCons with Python 3!
 
-	sudo apt install openocd
+We use [Doxypress][doxypress_binaries] to generate the API documentation:
 
-!!! warning "Ubuntus 'gcc-arm-none-eabi' package"
-	The Ubuntu package 'gcc-arm-none-eabi' causes [issues at the moment](https://github.com/modm-io/modm/issues/468) and ships only version 9.3.
-	We recommend using the toolchain provided by ARM:
+```sh
+sudo mkdir /opt/doxypress
+wget -O- https://download.copperspice.com/doxypress/binary/doxypress-1.4.0-ubuntu20.04-x64.tar.bz2 | sudo tar xj -C /opt/doxypress
+```
 
-		wget -O- https://developer.arm.com/-/media/Files/downloads/gnu-rm/10-2020q4/gcc-arm-none-eabi-10-2020-q4-major-x86_64-linux.tar.bz2 | sudo tar xj -C /opt/
-	
-	Add it to your systems PATH variable by appending the following line to `~/.bashrc`:
+Add the directory to your `PATH` variable in `~/.bashrc`:
 
-		export PATH="/opt/gcc-arm-none-eabi-10-2020-q4-major/bin:$PATH"
+```sh
+export PATH="/opt/doxypress:$PATH"
+```
 
-For Atmel/Microchip AVR targets:
 
-	wget -O- https://github.com/modm-io/avr-gcc/releases/download/v10.2.0/avr-gcc.tar.bz2 | sudo tar xj -C /opt
+#### Microchip AVR
 
-Add it to your systems PATH variable by appending the following line to `~/.bashrc`:
+Download and extract the [pre-built AVR toolchain][modm-avr-gcc]:
 
-	export PATH="/opt/avr-gcc/avr-gcc/bin:/opt/avr-gcc/avr-binutils/bin:$PATH"
+```sh
+wget -O- https://github.com/modm-io/avr-gcc/releases/download/v10.2.0/avr-gcc.tar.bz2 | sudo tar xj -C /opt
+```
 
-We recommend Doxypress to generate the documentation:
-Download the [binary][doxypress_binaries] for your distribution and unzip it:
+Add the `bin` directories to your `PATH` variable in `~/.bashrc`:
 
-	sudo mkdir /opt/doxypress
-	wget -O- https://download.copperspice.com/doxypress/binary/doxypress-1.4.0-ubuntu20.04-x64.tar.bz2 | sudo tar xj -C /opt/doxypress
+```sh
+export PATH="/opt/avr-gcc/avr-gcc/bin:/opt/avr-gcc/avr-binutils/bin:$PATH"
+```
```

Comment:
Why Microchip AVR above ARM? I would prefer to have Cortex-M instruction on top because it's probably the most frequently used target.